widgets/locate/FindAddress.js

```diff
@@ -120,6 +128,14 @@ define([
                 this.symbol.setStyle(SimpleMarkerSymbol.STYLE_DIAMOND);
                 this.symbol.setColor(new Color([255, 0, 0, 0.5]));
             }
+
+            if (this.map && !this.spatialReference) {
+                this.spatialReference = this.map.spatialReference.wkid;
+            }
+
+            if (!this.spatialReference) {
+                this.spatialReference = 26912;
+            }
```

Comment:
Possibly better way to do this:
```js
if (!this.spatialReference) {
  this.spatialReference = (this.map) ? this.map.spatialReference.wkid : 26912;
}
```

Also, `this.spatialReference` should probably be changed to something like `this.wkid` or `this.spatialReferenceWkid` so that it's not confused with being a reference to an `esri/SpatialReference` object.